streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -40,10 +40,11 @@ public KStreamBuilder() {
     }
 
     /**
-     * Creates a {@link KStream} instance from the specified topics.
+     * Creates a {@link KStream} instance for the specified topics.
      * The default deserializers specified in the config are used.
      *
      * @param topics    the topic names; must contain at least one topic name
+     * @return a {@link KStream} for the specified topics
```

Comment:
Question: What is the contract/the semantics, if any, for the order in which we turn multiple topics into a single stream?  IMHO we should be a bit more explicit here, which includes the option to say "We *do not* guarantee xyz..."

/cc @guozhangwang